### ISSUE TITLE
Deeplinking

### ISSUE THREAD

**Issue by JonEsparaz** [themeetinghouse/mobile#49]
It'd be pretty cool if we setup links so that if someone shares a teaching video, the link will open in the app (if the person has the app downloaded).

Not sure if this is feasible/possible.

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=588048700)

**Comment by georgevicbell** [comment]
Yup, we should be able to do it with expo. I believe the way it works on Jesus collective should make it work correctly.

Sent from my iPhone

On Aug 20, 2020, at 12:55 PM, Jon Esparaz <notifications@github.com> wrote:

﻿

It'd be pretty cool if we setup links so that if someone shares a teaching video, the link will open in the app (if the person has the app downloaded).

Not sure if this is feasible/possible.

**Comment by JonEsparaz** [comment]
For future reference: https://docs.expo.io/workflow/linking/#universal-links-on-ios

**Comment by GeorgeBellTMH** [comment]
- [x] add apple-app-site-association file to web (under public/.well-known) - it is redirecting everything /videos/*
- [x] Updated the app json to recognize the associatedDomains www.themeetinghouse.com, themeetinghouse.com and localhost:3006
- [x] update apple permissions
- [x] setup for android (in app.json, all /videos should forward)
- [ ] handle links

**Comment by GeorgeBellTMH** [comment]
Ok, it should work - it's not, but I assume some code needs to be written to handle things in the app... @JonEsparaz I leave it to you to figure out the rest...let me know if any issues...
